package.json

```diff
@@ -73,5 +73,8 @@
   },
   "scripts": {
     "postinstall": "chmod u+x .postinstall.js && node ./.postinstall.js"
+  },
+  "frontendDependenciesCSS": {
```

Comment:
I’m not sure if I’m happy with the name. I guess I would prefer something shorter like `bundleCSS` instead. Would do you think?